docs/pages/components/split-panel.md

```diff
@@ -239,16 +241,102 @@ To snap panels at specific positions while dragging, add the `snap` attribute wi
     transform: translateX(-3px);
   }
 
-  .split-panel-snapping-dots::before {
+  .split-panel-snapping .split-panel-snapping-dots::before {
     left: 100px;
   }
 
-  .split-panel-snapping-dots::after {
+  .split-panel-snapping .split-panel-snapping-dots::after {
     left: 50%;
   }
 </style>
 ```
 
+Or, if you want to snap the panel to every `100px` interval, as well as at 50% of the panel's size, you can use `snap="repeat(100px) 50%"`.
+
+```html:preview
+<div class="split-panel-snapping-repeat">
+  <sl-split-panel snap="repeat(16px) 50%">
```

Comment:
This didn't match the description above.

```suggestion
  <sl-split-panel snap="repeat(100px) 50%">
```

Reply:
Sorry about that, I changed the interval after testing because it wasn't immediately apparent that the 50% was adding an extra snap point, and I forgot to actually change it in the code.